packages/transport/src/transports/nodeusb.browser.ts

```diff
@@ -0,0 +1,10 @@
+import { WRONG_ENVIRONMENT } from './errors';
+
+export class NodeUsbTransport {
```

Comment:
Shouldn't be the name of this `NodeUsbBrowserTransport` to be consisten with the file name and to avoid confusion with `nodeusb.ts`?

What is the purpose of this? Just to let the developer know it is being used in the wrong environment?

Reply:
OK, I got it know, it is just a way to display error when baddly used.

Reply:
Yes, this is just replacement for NodeUsbTransport class in browser environment.

Reply:
nodeusb moved to another PR